tianshou/highlevel/params/alpha.py

```diff
@@ -32,5 +32,5 @@ def create_auto_alpha(
     ) -> tuple[float, torch.Tensor, torch.optim.Optimizer]:
         target_entropy = float(-np.prod(envs.get_action_shape()))
         log_alpha = torch.zeros(1, requires_grad=True, device=device)
-        alpha_optim = torch.optim.Adam([log_alpha], lr=self.lr)
+        alpha_optim = optim_factory.create_optimizer_for_params([log_alpha], self.lr)
```

Comment:
I prefer using kwargs (i.e. `lr=self.lr`), also safer against refactorings. Maybe force them in the new method?

Reply:
It is not reasonable to require kwargs for a two-arg method with two entirely different types.